src/Compatibility/Microsoft.DotNet.PackageValidation/ValidatePackage.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Diagnostics;
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.DotNet.ApiCompatibility;
+using Microsoft.DotNet.ApiCompatibility.Abstractions;
+using Microsoft.NET.Build.Tasks;
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.PackageValidation
+{
+    public class ValidatePackage : TaskBase
+    {
+        [Required]
+        public string PackagePath { get; set; }
+
+        public string RuntimeGraph { get; set; }
+
+        public string NoWarn { get; set; }
+
+        public bool RunApiCompat { get; set; }
+
+        public bool ValidateWithPreviousVersion { get; set; }
+
+        protected override void ExecuteCore()
+        {
+            if (!File.Exists(PackagePath))
```

Comment:
wondering if we should let the underlying IO operation throw instead to avoid the IO check here?

Reply:
yeah that should be fine. i checked package archive reader throws the similar exception